readme.md

```diff
@@ -158,9 +139,7 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** At least one of these will throw an error. Which one? How can you get around this issue?
 
-* **Your Answer:** 
-
----
+* **Your Answer:** I do not see any errors. (const response = await Series.find().select('_id title start_year season_count') )
```

Comment:
You should encounter an error when you run `.create().select()`. You will have to re-request the information or use the `.save()` method to return the updated record.